cmd/rqlited/main.go

```diff
@@ -132,6 +133,8 @@ func init() {
 	flag.IntVar(&compressionBatch, "compression-batch", 5, "Request batch threshold for compression attempt")
 	flag.StringVar(&cpuProfile, "cpu-profile", "", "Path to file for CPU profiling information")
 	flag.StringVar(&memProfile, "mem-profile", "", "Path to file for memory profiling information")
+	flag.StringVar(&sourceIp, "source-ip", "", "Specify a source ip address, when your node has multiple ip address segments")
```

Comment:
What do you mean by an "IP address segment"? Can you give me an example?

Reply:
Variable should be named `sourceIP`.

Reply:
"IP address segment"  When your host has multiple network segments, that is, multiple network cards, one of the address segments has a default route, but the other is the address that works as rqlite, that is, multiple address segments
```
eth0:  192.168.9.2/24  # Has a default route  
eth1:   172.16.9.2/24  #There is no default route, the policy is controlled by the "ip rule" command, rqlite works in this address segment
```